### DISCUSSION TITLE
Heroku env vars dont appear in the quasar ssr app

### DISCUSSION THREAD

**Discussion by eitan02581** [quasarframework/quasar#13482] [CLI - SSR mode]
i added env vars on heroku,
i deployed quasar SSR app
it seems that quasar ignores the configured env vars from heroku
what should I do?

**Comment by MilosPaunovic** [comment]
Any more details would be helpful... How are you using 'em in Quasar, some code reproduction...

**Comment by eitan02581** [comment]
Hey,
on Heroku i configured some env vars. on another node js that contains my backend it works properly..
![her](https://user-images.githubusercontent.com/36443745/169611431-58651920-4ea1-4e9a-8142-1390f0969fbe.PNG)

on quasar side i have an envparser.js file that parse my local env file.  
![code](https://user-images.githubusercontent.com/36443745/169611709-e458c4b5-c80b-48fb-8556-0832c244ae35.PNG)

added also the quasar conf file -
![conf](https://user-images.githubusercontent.com/36443745/169611814-59b5e779-6b6c-49e1-ae9c-91cef452e119.PNG)




thanks

**Reply by MilosPaunovic** [reply]
Well, `envparser()`, as I understand, parses your local `.env` files. But, without any code reproduction from your side, it's kinda imposible to provide any help as this way is just working in the dark. Create a minimal reproduction repository and share it with us, so someone could give it a closer look.

**Comment by suleimanas** [comment]
```
env: {
...envparser(),
GOOGLE_API_KEY: process.env.GOOGLE_API_KEY,
...
}
```
